README.md

```diff
@@ -0,0 +1,5 @@
+MFA Credential Auto Updater for AWS
```

Comment:
add 

* installation guide
* requirements.txt

if you want extra credit, you can call fabric dynamically so that you don't need to run fab get_mfa_credentials, you can just run get_mfa_credentials, add a setup.py, make get_mfa_credentials an entrypoint, then bundle everything and upload to PyPI.

Reply:
I think I'll do this in a different branch